content/events/2020/02/2020-02-25-dap-learning-series-justifying-web-analysts.md

```diff
@@ -1,5 +1,5 @@
 ---
-# View this page at https://digital.gov/event/2020/02/dap-learning-series-justifying-web-analysts
+# View this page at https://digital.gov/event/2020/03/dap-learning-series-justifying-web-analysts
```

Comment:
How did this change? Curious if this was just done with the workflow tool, or if this was done by hand.

Reply:
this was done via the workflow tool - we added speakers and tags to the page as we are diving deep into the meta-tagging process

Reply:
ok!